A/ASTInterpreter2/Compat.toml

```diff
@@ -1,3 +1,3 @@
 [0]
 DebuggerFramework = "0.1.1-0"
-julia = "0.6-0"
```

Comment:
Note that ASTInterpreter2 got close (on the `master` branch) to working on 0.7/1.0, but there was never a release.